### PR TITLE
Show not paid bookings with account payment method

### PR DESCRIPTION
**Pull request by Raibo**

### CODE REVIEW

File: Api/Services/Accommodations/Bookings/BookingRecordsManager.cs [hunk 1]
```diff
@@ -200,7 +201,8 @@ public async Task<Result<List<SlimAccommodationBookingInfo>>> GetAgentBookingsIn
         {
             var bookingData = await _context.Bookings
                 .Where(b => b.AgentId == agentContext.AgentId)
-                .Where(b => b.PaymentStatus != BookingPaymentStatuses.NotPaid)
+                .Where(b => b.PaymentMethod == PaymentMethods.BankTransfer
```
Comment (mingazhev) [inline Api/Services/Accommodations/Bookings/BookingRecordsManager.cs:209]: Should there be the brackets?

Comment (Raibo) [reply Api/Services/Accommodations/Bookings/BookingRecordsManager.cs:209]: No, it's good like this.
The 'and' (&&) operation occurs earlier than 'or' (||) operation, like '*' and '+'. 
A statement like this `var testBool = true || false && false;` ends up true.

Comment (mingazhev) [reply Api/Services/Accommodations/Bookings/BookingRecordsManager.cs:209]: Anyway, this is hard to read and understand the logic at one glance

Comment (Raibo) [reply Api/Services/Accommodations/Bookings/BookingRecordsManager.cs]: Added brackets and newline.

**Review by mingazhev — APPROVED** [review]

**Pull request merged**